src/gateway/kong-enterprise/index.md

```diff
@@ -1,6 +1,86 @@
+
+## Dev Portal
+
+Streamline developer onboarding with the [Dev Portal](/gateway/{{page.kong_version}}/kong-enterprise/developer-portal/), which offers a self-service developer experience to discover, register, and consume published services from your Service Hub catalog. This customizable experience can be used to match your own unique branding and highlights the documentation and interactive API specifications of your services. Enable application registration to automatically secure your APIs with a variety of authorization providers.
+
+## Audit logging
+
+{{site.base_gateway}} provides a granular logging facility on its Admin API. This
+allows cluster administrators to keep detailed track of changes made to the
+cluster configuration throughout its lifetime, aiding in compliance efforts and
+providing valuable data points during forensic investigations. Generated audit
+log trails are [Workspace](/gateway/{{page.kong_version}}/admin-api/workspaces/reference) and [RBAC](/gateway/{{page.kong_version}}/admin-api/rbac/reference)-aware,
+providing {{site.base_gateway}} operators a deep and wide look into changes happening within
+the cluster.
+
+## Keyring and data encryption
+
+{{site.base_gateway}} allows you to store sensitive data fields, such as consumer secrets, in an encrypted format within the database. This provides encryption-at-rest security controls in a {{site.base_gateway}} cluster.
+
+This functionality provides transparent, symmetric encryption of sensitive data fields at rest. Transparency refers to the fact that, when enabled, encryption/decryption of data is done on-the-fly by {{site.base_gateway}} immediately before writing/immediately after reading from the database. Responses generated by the Admin API containing sensitive fields continue to show data as plaintext, and runtime elements of {{site.base_gateway}} (such as plugins) that require access to sensitive fields do so transparently, without requiring additional configuration.
+
+See [Keyring and data encryption](/gateway/{{page.kong_version}}/kong-enterprise/db-encryption/) for more information. 
+
+## Roles-based access control (RBAC)
+
+You can configure {{site.base_gateway}}'s [RBAC](/gateway/{{page.kong_version}}/admin-api/rbac/reference/) feature using the
+[Admin API](/gateway/{{page.kong_version}}/admin-api/) or [Kong Manager](/gateway/{{page.kong_version}}/configure/auth/rbac). RBAC allows you to configure the roles and permissions for admin users to access resources in {{site.base_gateway}}.
+
+There are 4 basic entities involving RBAC.
+
+- **User**: The entity interacting with the system. Can be associated with
+  zero, one or more roles.
+- **Role**: Set of permissions. Has a name and can be associated with zero, one or more permissions.
+- **role_endpoint**: A set of enabled or disabled actions (`read`, `create`, `update`, `delete`) for `endpoint`.
+- **role_entity**: A set of enabled or disabled actions (`read`, `create`, `update`, `delete`) for `entity`.
```

Comment:
```suggestion
```
Remove this whole thing. Too much info for the intro, plus it's specific to the admin api.